### PR TITLE
[WIP] add web_disable_export

### PR DESCRIPTION
**Pull request by luc-demeyer**
this module removes the 'Export' option from the 'More' menu in the list view except for the admin user.

### CODE REVIEW

File: web_disable_export/static/src/js/disable_export.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+openerp.web_disable_export = function(instance) {
+
+    var _t = instance.web._t;
+    var SUPERUSER_ID = 1;
+
+    instance.web.Sidebar.include({
+
+        add_items: function(section_code, items) {
+            // allow Export for admin user
+            if (this.session.uid == SUPERUSER_ID) {
```
Comment (nhomar) [inline web_disable_export/static/src/js/disable_export.js:33]: It should be great if it is checked by ACL and not only by SUPERUSER_ID, don't you think?

Comment (luc-demeyer) [inline web_disable_export/static/src/js/disable_export.js:33]: Nhomar,

You are right but the ‘sponsoring’ customer only wanted to pay a couple of hours to disable the export and hence we implemented this. 

Regards,

Luc

 http://www.noviat.com/ www.noviat.com

Rusatiralaan 1, 1083 Brussel

+32 2 808 86 38

**Comment by pedrobaeza** [comment]
Extract the description to a README.rst file and please put what Nhomar comments in a TODO section of this description.

**Comment by pedrobaeza** [comment]
Please remove description key from manifest file.

**Comment by Alembo** [comment]
This can also be done with http://change-openerp-web.blogspot.com/2013/08/hide-export-in-more-option-in-openerp-7.html

Yours would be better if we could also add a expect group instead of only user admin.

File: web_disable_export/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    'license': 'AGPL-3',
+    'author': 'Noviat',
+    'website': 'http://www.noviat.com',
+    'category': 'Web',
+    'description': """
+
+Remove the 'Export' option from the 'More' menu
+in the list view except for the admin user
+
+""",
```
Comment (dreispt) [inline web_disable_export/__openerp__.py:35]: The description key can be removed since the README will be automatically used for it

**Comment by hbrunn** [comment]
I think this should happen on template level and use a group that is allowed to export to be useful :-1:

**Comment by luc-demeyer** [comment]
@hbrunn 
I agree that your suggestion would make it better, but that goes beyond what the sponsoring customer wanted to pay for.

**Comment by legalsylvain** [comment]
Hi, 
I understand @luc-demeyer PoV. 
By the other side, remarks of @hbrunn and @nhomar are very relevant.

Maybe we can mark this PR as 'WIP' and 'help Wanted' and wait some other contributors to finish this module ?
Regards.

**Comment by luc-demeyer** [comment]
> > > Maybe we can mark this PR as 'WIP' and 'help Wanted' 
> > > Sounds ok to me. In the meanwhile I'll publish it via our apps.odoo.com branch (and try not to forget to remove it once it becomes available via the OCA).

**Comment by hbrunn** [comment]
closing due to inactivity

**Pull request closed without merging**